app/js/lib/foxbox/services/motion-sensor.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+import BaseService from './base';
+
+const TYPE = 'motion-sensor';
+
+export default class MotionSensorService extends BaseService {
+  constructor(props, config) {
+    super(props, config);
+    Object.seal(this);
+  }
+
+  get type() {
+    return TYPE;
+  }
```

Comment:
Yep! (that's not my laziness, I just feel API will change again, that's why I'm trying to implement only something that is needed :))